x-pack/plugins/search_indices/public/components/details_page/details_page.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiPageSection, EuiSpacer, EuiButton, EuiPageTemplate } from '@elastic/eui';
+import React, { useMemo } from 'react';
+import { FunctionComponent } from 'react';
+import { RouteComponentProps } from 'react-router-dom';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useIndex } from '../../hooks/api/use_index';
+import { i18n } from '@kbn/i18n';
+
+export const SearchIndexDetailsPage: FunctionComponent<
+  RouteComponentProps<{ indexName: string }>
+> = ({ location: { search } }) => {
+  const queryParams = useMemo(() => new URLSearchParams(search), [search]);
```

Comment:
IMO we should make the index name a route parameter and not a query parameter. Route params are required but query parameters are optional.